addons/hr_recruitment/models/hr_recruitment.py

```diff
@@ -283,6 +283,7 @@ def _compute_date_closed(self):
     @api.model
     def create(self, vals):
         if vals.get('department_id') and not self._context.get('default_department_id'):
+            # VFE FIXME remove this context and support batch record creation...
```

Comment:
The context fixmes are there for the responsible teams, because it requires functional knowledge to be sure the context removal doesn't break the existing behavior.

But if you don't like fixme's, then I can remove them :thinking:.

Reply:
It's ok for the dev but we should not add new TODO in "cleanup" branches.